### ISSUE TITLE
Illegal reflective access operations

### ISSUE THREAD

**Issue by dsubelman** [jqno/equalsverifier#385]
**Describe the bug**
When verifying a class that imports DoubleSummaryStatistics (or the class itself) it throws an illegal reflective access warning for field java.util.DoubleSummaryStatistics.count

**To Reproduce**
Verify class DoubleSummaryStatistics.

**Code that triggers the behavior**
`EqualsVerifier.simple().forClass(DoubleSummaryStatistics.class).verify();`

**Error message**
```
[ERROR] WARNING: An illegal reflective access operation has occurred
[ERROR] WARNING: Illegal reflective access by nl.jqno.equalsverifier.internal.reflection.FieldModifier (file:/Users/dsubelman/.m2/repository/nl/jqno/equalsverifier/equalsverifier/3.5.2/equalsverifier-3.5.2.jar) to field java.util.DoubleSummaryStatistics.count
[ERROR] WARNING: Please consider reporting this to the maintainers of nl.jqno.equalsverifier.internal.reflection.FieldModifier
[ERROR] WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[ERROR] WARNING: All illegal access operations will be denied in a future release
```

**Expected behavior**
No warning

**Version**
3.5.2

**Additional context**

**Comment by jqno** [comment]
I've released v3.5.3 which adds Prefab values for `DoubleSummaryStatistics`, `IntSummaryStatistics` and `LongSummaryStatistics`

**Comment by dsubelman** [comment]
Wow, that was fast. Thanks!
